src/controllers/action-queue/types.ts

```diff
@@ -173,7 +161,7 @@ export type ActionProgramCompleteMessage = { title: string; message: string }
 
 export type ActionProgram = {
   programId: string
-  actionOp: ActionOp
+  actionOp: NodeActionOp
```

Comment:
Oh you did it. Awesome; cool cool

Reply:
Erm, needs revert cause it cascades type changes for the rest of the runtime implementation. Let's postpone this work item.